requests_unixsocket/__init__.py

```diff
@@ -1,19 +1,32 @@
-import requests
 import sys
 
+import requests
+from requests.compat import urlparse, unquote
+
 from .adapters import UnixAdapter
 
-DEFAULT_SCHEME = 'http+unix://'
+
+def default_urlparse(url):
+    return UnixAdapter.Settings.ParseResult(
```

Comment:
Seems a bit of a waste to urlparse url three times

how about:

```
parsed = urlparse(url)
return UnixAdapter...
```

Reply:
Yep, you're right. I will definitely update that. Thanks!

Reply:
Updated. Thanks again!